.github/dependabot.yml

```diff
@@ -0,0 +1,26 @@
+version: 2
+updates:
+  # Dependencies listed in go.mod
+  - package-ecosystem: "gomod"
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
+    commit-message:
+        prefix: ":seedling:"
```

Comment:
Would it be a good idea to include `include: "scope"` in `commit-message` to get a list of dependencies updated in the commit? Reference: [commit-message](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#commit-message)

Reply:
+1 

Also CAPZ doesn't use the "seedling"  prefix like CAPI does, we just the k8s release note generator tool which uses the labels to classify release notes